scrapperFunctions/index.js

```diff
@@ -98,13 +136,35 @@ ScrapPage.prototype.expectedRows = async function (select) {
 };
 
 /**
- * Collect the data of different screens
- * @param {string} TABLE_SELECTOR The identifier of the table
- * @param {string} ROW_SELECTOR How can I find the number of rows in the table
- * @param {string} nextPageButton The identifier of the button
- * @param {int | double} time How much time wait for do the process again
- * @param {function} callback A function needed in progressBar
- * @returns Array[Array[string]]
+ * This asynchronous function uses the other functions to bring the
```

Comment:
witch another functions ?

Reply:
expectedRows, getDataTable and clickButton